apps/src/code-studio/DisabledBubblesAlert.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { PropTypes } from 'react';
+import Alert from '@cdo/apps/templates/alert';
+import i18n from '@cdo/locale';
+
+const styles = {
+  bold: {
+    fontFamily: '"Gotham 5r", sans-serif'
+  }
+};
+
+/**
+ * Component that displays a small notification at the top of the page when
+ * postMilestone is disabled (and thus progress is disabled)
+ */
+const DisabledBubblesAlert = React.createClass({
+  propTypes: {
+    isHocScript: PropTypes.bool.isRequired
+  },
+
+  getInitialState() {
+    // Once alert has been dismissed, don't show again.
+    const disabledBubblesAlertSeen = sessionStorage.getItem('disabledBubblesAlertSeen');
```

Comment:
[Wiki page updated](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) to mention the current origin.